src/components/AddBook.js

```diff
@@ -1,17 +1,37 @@
-import React from 'react';
+import { useState } from 'react';
 
-const AddBook = () => (
-  <>
-    <div className="add-container">
-      <h2>ADD NEW BOOK</h2>
-      <form action="#" method="post">
-        <input type="text" placeholder="Book Title" />
```

Comment:
- [ ] let's use `useDispatch` for adding new book
- [ ] In your `Books` components, to display list of books use `useSelector` to consume state.

Reply:
@uwadonat Thank you. this is wrong PR for review